src/components/chip-group/chip-group.tsx

```diff
@@ -0,0 +1,199 @@
+import {
+  Component,
+  h,
+  VNode,
+  Prop,
+  Element,
+  Listen,
+  EventEmitter,
+  Event,
+  Host,
+  Watch
+} from "@stencil/core";
+import { focusElementInGroup } from "../../utils/dom";
+import { InteractiveComponent, updateHostInteraction } from "../../utils/interactive";
+import { createObserver } from "../../utils/observers";
+import { Scale, SelectionMode } from "../interfaces";
+
+/**
+ * @slot - A slot for adding one or more `calcite-chip`.
+ */
+@Component({
+  tag: "calcite-chip-group",
+  styleUrl: "chip-group.scss",
+  shadow: true
+})
+export class ChipGroup implements InteractiveComponent {
+  //--------------------------------------------------------------------------
+  //
+  //  Element
+  //
+  //--------------------------------------------------------------------------
+
+  @Element() el: HTMLCalciteChipGroupElement;
+
+  //--------------------------------------------------------------------------
+  //
+  //  Public Properties
+  //
+  //--------------------------------------------------------------------------
+
+  /** When true, interaction is prevented and the component is displayed with lower opacity. */
+  @Prop({ reflect: true }) disabled = false;
+
+  /** Specifies the size of the component. */
+  @Prop({ reflect: true }) scale: Scale = "m";
+
+  /** Specifies the selection mode of the component. */
+  @Prop({ reflect: true }) selectionMode: Extract<
+    "multiple" | "single" | "single-persist" | "none",
+    SelectionMode
+  > = "none";
+
```

Comment:
We might need a required `label` prop (details below on why):

```ts
  /** Accessible name for the component. */
  @Prop() label!: string;
```

Reply:
Will add - will each child chip then use `aria-labelledby` to reference the parent label? No need to reference an `id` at all?

In the case a `calcite-chip` will not be wrapped in a `calcite-chip-group` (still valid use case), we won't need to set any of: `role`, `aria-checked`, `aria-labelledby`, correct?

Reply:
We shouldn't need any additional context, the label provides context when first entering the `chip-group`, and context of the child `chip`s and if it selected, or not selected.

Reply:
Sorry, I meant when a `calcite-chip` is just placed solo on a page, not within a `calcite-chip-group` (so no selectionMode is assigned) - we just don't need to set any of these aria items, correct?

Reply:
Some of the basic configurations seem to work well as-is, but some of the more complex ones with icons, icons and close buttons, just images, and when used in cards might need some additional support. There's some context missing in those types in both JAWS and NVDA. Will try to diagnose later on this evening.